app/views/layouts/newflow_layout.html.erb

```diff
@@ -25,6 +25,19 @@
             <%= yield %>
         </div>
 
+        <div class="content" id="newflow-footer">
+            <div>
+                <%= link_to "http://www.rice.edu" do %>
+                    <%= image_tag 'rice_logo_4.png', id: "footer-rice-logo-image" %>
+                <% end %>
+            </div>
+
+            <div id="newflow-footer--copyright">
+                <%= link_to OpenStax::Utilities::Text.copyright('2013', COPYRIGHT_HOLDER), main_app.copyright_path %> |
```

Comment:
hard-coded year? can this be done programmatically?

Reply:
Yes, the start year is fine. It'll always be the same.